### PR TITLE
Intercom integration

### LINKED ISSUE #588: Add Intercom widget (https://www.intercom.com/) or similar to Mosaic to make it easier for participants to get in touch with us
- this would just be to test this out -- we're uncertain that it's actually a good idea to make it this easy to get in touch with Ben R/experiment runners
- note that our lag team in responding would often be like 12 hrs, so we need to choose something that makes sense for that sort of lag time (e.g. email notifications if there's a long lag)
- unclear how important this is

### PR DESCRIPTION
**Pull request by andrewschreiber**
closes #588. Reach out to Andrew Schreiber or Ben Rachbach for login information. Currently the system tracks chat users via Intercoms built in functionality, which allows chat users to enter their email. A future expansion could automatically include emails for logged in users. The Intercom widget should appear on all pages within Mosaic.

### CODE REVIEW

**Comment by andrewschreiber** [comment]
Chat inside Mosaic:

image

Using the Intercom admin inbox:

image

File: client/public/index.html [hunk 1]
```diff
@@ -19,24 +22,99 @@
...
+        g.identifyAccount = function(i, v) {
+          o = "account";
+          v = v || {};
+          v.acctId = i;
+          g(o, v);
+        };
+        g.clearUserCookie = function() {};
+      })(window, document, window["_fs_namespace"], "script", "user");
+    </script>
+    <script>
```
Comment (andrewschreiber) [inline client/public/index.html]: Prettier automatically reformatted some of index.html. The Intercom additions are between line 86 and 117.

File: client/src/App.tsx [hunk 2]
```diff
@@ -121,6 +128,10 @@ const Routes = () => (
 
 class App extends React.Component {
   public render() {
+    window.Intercom("boot", {
+      app_id: "gmkvd6s1",
```
Comment (zjmiller) [inline client/src/App.tsx]: Maybe put this in `config.js`? Not a big deal either way.

Comment (andrewschreiber) [reply client/src/App.tsx]: Looks like we don't have a config.js/config.json for the client. Seems like a candidate for a future refactor.

**Pull request merged**